api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -79,6 +79,74 @@ message UIRecoveryCodesPrintClickEvent {
   string username = 1;
 }
 
+// Contains common metadata for Discover related events.
+message DiscoverMetadata {
+  // Uniquely identifies Discover wizard "session". Will allow to correlate
+  // events within the same Discover wizard run. Can be UUIDv4.
```

Comment:
Can it be stuff other than an UUIDv4?

Reply:
:pray: 
Removing implementation details.